### PR TITLE
Melnyk/US-875_FilterEventsOnTheMapByCategoriesPageObject

### PR DESCRIPTION
**Pull request by MelnykNikita**
Reorganized US_FilterEventsOnTheMapByCategories to Page Object.
Added EventsPage. 
Added WebElement events and method clickEvents() in CategoriesPage.
Added WebElement categories and method clickCategories() in MapHeadPageForGuestUserPage.
Added 2 classes with Tests:
- FilterEventsOnTheMapByCategoriesPageObjectWithChrome
- FilterEventsOnTheMapByCategoriesPageObjectWithFirefox

### CODE REVIEW

**Review by MaxKovalets — APPROVED** [review]

**Review by JugerJuger — COMMENTED** [review]
I think you added, the best way to use waiters!!! You must share this solution with all team,  tomorrow.

**Review by JugerJuger — APPROVED** [review]

**Review by TyshchenkoRoma — APPROVED** [review]

File: src/test/java/rv/atqc/gttesting/archex/CategoriesPage.java [hunk 1]
```diff
@@ -18,5 +20,8 @@ public TracksPage clickTracks(){
 		tracks.click();
 		return new TracksPage(driver);
 	}
-
+	public EventsPage clickEvents(){
+		events.click();
+		return new EventsPage(driver);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/CategoriesPage.java]: Why not just return this?

File: src/test/java/rv/atqc/gttesting/archex/EventsPage.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    public WebElement getGameFilter(){
+        return gamePlaceFilter;
+    }
+    public WebElement getFestivalFilter(){
+        return festivalPlaceFilter;
+    }
+    public WebElement getMeetingFilter(){
+        return meetingPlaceFilter;
+    }
+    public WebElement getGameGreenIcon(){ return gameGreenIcon; }
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/EventsPage.java]: format please

File: src/test/java/rv/atqc/gttesting/archex/MapHeadPageForGuestUserPage.java [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+	@FindBy(how = How.XPATH, using = "html/body/main/div/place-list/div/div/div[1]/div[2]/div/div[3]/button")
+	private WebElement categories;
+
+	public MapHeadPageForGuestUserPage(WebDriver driver) {
 		super(driver);
 	}
 
+	public CategoriesPage clickCategories(){
+		categories.click();
+		return new CategoriesPage(driver);
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/MapHeadPageForGuestUserPage.java]: Why not just return this?

**Review by dmatsev — APPROVED** [review]
approve but need to merge changes for AbstarctPage with waiters

**Pull request merged**